week3/Borislav/issue_TAPC-3.py

```diff
@@ -0,0 +1,46 @@
+# -*-config: utf_8 -*-
+
+
+# The function of which we read docstring
+def som_func():
+    """
+    Calculate the square root of a number.
+
+    Accepts:
+    n: the number to get the square root of.
+    Returns:
+    the square root of n.
+    Raises:
+    TypeError: if n is not a number.
+    ValueError: if n is negative.
+    """
+    pass
+
+
+def doc_validation(func):
+    """"
+    Function decorator
+    Records the results of the validation of the docstring in the log file
+    log.txt
+    """
+    def wrapper_function(*args, **kwargs):
+
+        doc_str = str(args)
+        if (doc_str.count('Accepts:') > 0) and (doc_str.find('Raises:') > 0) and (doc_str.find('Returns:') > 0):
```

Comment:
Why do you use string.count ?
Why do you use string.find and not `in` ?

Reply:
Good qestion. This was a test. Sory for that. I'll check  the code and commit changes when i have more time. Best regards.